litellm/integrations/opentelemetry.py

```diff
@@ -115,6 +115,16 @@ def __init__(
             otel_exporter_logger = logging.getLogger("opentelemetry.sdk.trace.export")
             otel_exporter_logger.setLevel(logging.DEBUG)
 
+        # Don't override the tracer provider set by `opentelemetry-instrument`
```

Comment:
Maybe just "Don't override a tracer provider already set by the user". While opentelemetry-instrument is one way to set it and very common, it isn't the only one I think.

Reply:
good idea will change